pkg/controller/common/helpers.go

```diff
@@ -1375,3 +522,180 @@ func GetBootstrapAPIServer() (*configv1.APIServer, error) {
 	}
 	return apiserver, nil
 }
+
+const (
+	// osLabel is used to identify which type of OS the node has
```

Comment:
suggestion (non-blocking): It might be good to keep capitalization consistent between the code & comment addressing it (so `osLabel` to `OSLabel` in this comment).